app/league/all/page.tsx

```diff
@@ -18,6 +19,11 @@ const Leagues = (): JSX.Element => {
   const [leagues, setLeagues] = useState<ILeague[]>([]);
   const [currentWeek, setCurrentWeek] = useState<IGameWeek['week']>(1);
   const { user } = useDataStore((state) => state);
+  const { getUser } = useAuthContext();
+
+  if (!user.id || user.id === '') {
```

Comment:
This should go into the AuthContextProvider. Otherwise, every page will need this code.

Reply:
✅ Done.